pom.xml

```diff
@@ -98,6 +98,7 @@
     <module>hadoop</module>
     <module>integration/spark-common</module>
     <module>integration/spark-common-test</module>
+    <module>integration/presto</module>
```

Comment:
Add integration/presto module to parent pom.xml can be consider later once all issues be fixed and all test cases of presto be added.
So please remove it from your PR currently